test/images/scripts/lib/debian.install

```diff
@@ -46,6 +46,9 @@ if [ -n "$do_build" ]; then
     tar -xzf cockpit-*.tar.gz
     ( cd cockpit-*/
       cp -rp tools/debian debian
+      # Hack: Remove PCP build dependencies while pcp is not in testing
+      # (https://tracker.debian.org/pcp)
+      sed -i '/libpcp.*-dev/d' debian/control
```

Comment:
I like that. But lets take it a step further:

 1. if ```-s``` or ```RELEASE_DEBIAN``` is a directory, then use it directly as a ```debian``` directory.
 2. If ```-s``` or ```RELEASE_DEBIAN``` points to an executable, then execute it to produce a ```debian``` directory. 

That said, in either case, how would we pass in information about which DISTRIBUTION we're creating an DSC for? Also currently we create one DSC and use it for multiple distributions, and then munge it further in ```release-debian``` ... however I'm not against creating multiple DSC files that are distribution appropriate right from ```release-dsc```.